File: LearningPython/ShinKattisProblems/sibice.py


line = input().split()
a, b, c = int(line[0]), int(line[1]), int(line[2])
size = (b**2+c**2)**0.5
ans = []
for i in range(a):
    num = int(input())
    if num <= size:
        ans.append("DA")
    else:
        ans.append("NE")
for x in ans:
    print(x)

File: LearningPython/ShinKattisProblems/sibice.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
  int a, b, c, i;
  cin >> a >> b >> c;
  double size = sqrt(b*b+c*c);
  int num;
  string ans[1000] = {};
  for(i=0; i<a; i++)
  {
    cin >> num;
    if(num <= size) ans[i] = "DA";
    else ans[i] = "NE";
  }
  for(i=0; i<a; i++) cout << ans[i] << endl;

  return 0;
}
